doc/tutorials/acrn_configuration_tool.rst

```diff
@@ -122,33 +122,34 @@ Additional scenario XML elements:
   Specify the bitmap of consoles where logs are printed.
 
 ``LOG_BUF_SIZE`` (a child node of ``DEBUG_OPTIONS``):
-  Specify the capacity of log buffer for each physical CPU.
+  Specify the capacity of the log buffer for each physical CPU.
 
 ``RELOC`` (a child node of ``FEATURES``):
-  Specify whether enable hypervisor image relocation when boot.
+  Specify whether hypervisor image relocation is enabled on booting.
 
 ``SCHEDULER`` (a child node of ``FEATURES``):
-  Specify the CPU scheduler to be used by the hypervisor.
+  Specify the CPU scheduler used by the hypervisor.
   Supported schedulers are: ``SCHED_NOOP``, ``SCHED_BVT`` and ``SCHED_IORR``.
 
 ``MULTIBOOT2`` (a child node of ``FEATURES``):
-  Specify whether support boot ACRN hypervisor image from multiboot2 protocol.
-  ACRN hypervisor support boot from multiboot protocol always.
+  Specify whether ACRN hypervisor image is booted using multiboot2 protocol.
+  By default, ACRN hypervisor is booted using multiboot protocol.
 
```

Comment:
hi David,
The boot protocol is decided by bootloader.
In de-privilege mode ACRN hypervisor boot from a integrated internal bootloader in acrn.efi with multiboot protocol, whereas in direct boot mode, ACRN hypervisor boot from external bootloader(SBL/ABL/GRUB). SBL and ABL support multiboot only, but GRUB support both. User could choose multiboot or multiboot2 protocol to boot hypervisor in GRUB menu.
In current code, hypervisor provide support for multiboot always whatever CONFIG_MULTIBOOT2 is, and only support multiboot2 when "MULTIBOOT2=y". With this implementation, user must set "MULTIBOOT2=y" when he want to boot hypervisor from GRUB multiboot2, otherwise he can only boot hypervisor from ABL/SBL/acrn.efi or GRUB multiboot.

Reply:
OK, I simplified that in another revision.